src/subscription_manager/exceptions.py

```diff
@@ -74,7 +75,14 @@ def format_bad_ca_cert_exception(self, bad_ca_cert_error, message_template):
         return message_template % bad_ca_cert_error.cert_path
 
     def format_ssl_error(self, ssl_error, message_template):
-        return message_template % str(ssl_error)
+        # SSL error has fixed form: [SSL/BIO/...: ERRORO_ID] readable string (file.c:#line)
```

Comment:
This is hacky, imagine what happens if OpenSSL re-formats its error messages, for example, if they were to emit something like `[SSL: SSLV3_ALERT_CERTIFICATE_UNKNOWN] sslv3 alert certificate unknown [subject=CN=example.com] (_ssl.c:579)`, then the regex produces the empty string. We can't really predict how SSL handling might change. Already we are at the mercy of two libraries: python standard library `ssl` and `m2crypto`. Let's not couple ourselves to any particular SSL implementation anymore than we have to.